server/src/main/resources/rules/rules.js

```diff
@@ -1400,6 +1416,38 @@ var Entitlement = {
         return this.build_func("do_pre_physical_only")();
     },
 
+    // pre_virt_only already covers is guest
+    do_pre_unmapped_guests_only: function(context, result) {
+        var caller = context.caller;
+        var consumer = context.consumer;
+        var unmapped_guest_pool = Utils.equalsIgnoreCase('true', context.getAttribute(context.pool, UNMAPPED_GUESTS_ONLY));
+
+        if (unmapped_guest_pool) {
+            if (context.hostConsumer){
+                if (BEST_POOLS_CALLER == caller ||
+                    BIND_CALLER == caller) {
+                    result.addError("virt.guest.cannot.use.unmapped.guest.pool.has.host");
+                }
+                else {
+                    result.addWarning("virt.guest.cannot.use.unmapped.guest.pool.has.host");
```

Comment:
I'm not sure about the purpose of this.  @wottop will have to answer, but I see that construction in a couple of other places (like `do_pre_cores`).  I'm guessing it's to mark the system yellow when the system itself is getting its compliance status?